pkg/kubelet/kubelet.go

```diff
@@ -1305,13 +1313,17 @@ func (kl *Kubelet) RlimitStats() (*statsapi.RlimitStats, error) {
 // 3.  the plugins directory
 // 4.  the pod-resources directory
 // 5.  the checkpoint directory
+// 6.  the pod logs root directory
 func (kl *Kubelet) setupDataDirs() error {
 	kl.rootDirectory = filepath.Clean(kl.rootDirectory)
 	pluginRegistrationDir := kl.getPluginsRegistrationDir()
 	pluginsDir := kl.getPluginsDir()
 	if err := os.MkdirAll(kl.getRootDir(), 0750); err != nil {
 		return fmt.Errorf("error creating root directory: %v", err)
 	}
+	if err := os.MkdirAll(kl.getPodLogsDir(), 0755); err != nil {
+		return fmt.Errorf("error creating pod logs root directory: %v", err)
```

Comment:
include the directory name into the error message please

Reply:
Updated